src/main/java/com/cinema/model/MovieSession.java

```diff
@@ -0,0 +1,33 @@
+package com.cinema.model;
+
+import java.time.LocalDateTime;
+import javax.persistence.Entity;
+import javax.persistence.FetchType;
+import javax.persistence.GeneratedValue;
+import javax.persistence.GenerationType;
+import javax.persistence.Id;
+import javax.persistence.ManyToOne;
+import javax.persistence.Table;
+import lombok.EqualsAndHashCode;
+import lombok.Getter;
+import lombok.NoArgsConstructor;
+import lombok.Setter;
+import lombok.ToString;
+
+@Getter
+@Setter
+@NoArgsConstructor
+@ToString(of = {"id", "movie", "cinemaHall", "showTime"})
+@EqualsAndHashCode(of = {"id", "movie", "cinemaHall", "showTime"})
```

Comment:
google about Data annotation, that wil make list of annotation shorter

Reply:
done